### DISCUSSION TITLE
Editing Skeletons

### DISCUSSION THREAD

**Discussion by AnonymousDevelopel** [AndreaCatania/RuntimeSkeletalMeshGenerator#19] [Q&A] [answered]
Is there a way to delete and rename bones with this system?

**Comment by Fransferdy** [comment, marked as answer]
I think it is possible. You would have to create a copy of that skeleton with your edits, and fix each vertex individually(to stop using the deleted bones), then create the mesh asset with the new skeleton and return it.
